synapse/lib/cell.py

```diff
@@ -563,6 +563,12 @@ async def fini():
             'cell': self
         }
 
+    async def anit2(self):
+        '''
+        This must be called near the end of subclass initialization
+        '''
+        await self.nexsroot.recover()
```

Comment:
must every cell.__anit__ implementation call this?

Reply:
Everyone that uses a nexslog

Reply:
This method name needs work.  Are we trying to create a spot for all "after you've initialized" stuff?

Reply:
ok

Reply:
What if we did `postBaseAnit()` (which defaulted to `pass`), which was called inside of `Base.anit()`, which the `Cell` defines as `postCellAnit()`, which then calls this `recover()` routine. This would ensure that *all* Cells are covered with this, since all Cell's could use the nexus for doing user change control.

Reply:
Discussed over voice and resolved.